node/node_test.go

```diff
@@ -49,7 +49,7 @@ func TestNodeStartStop(t *testing.T) {
 	require.NoError(t, err)
 	select {
 	case <-blocksSub.Out():
-	case <-blocksSub.Cancelled():
+	case <-blocksSub.Canceled():
 		t.Fatal("blocksSub was cancelled")
```

Comment:
ditto

Reply:
seen a lot of `cancelled` in the logger or error messages. Should It be aligned to `canceled`?

Reply:
Aesthetically speaking, yes. 🙂 But those weren't part of the PR this is backporting, so I'd rather not grow the diff for purely cosmetic reasons.